test/support/test_repo.exs

```diff
@@ -14,14 +15,19 @@ defmodule NebulexEcto.TestAdapter do
     {:ok, []}
   end
 
-  def child_spec(_repo, opts) do
-    :nebulex_ecto = opts[:otp_app]
+  def init(opts) do
+    :ecto_nebulex = opts[:otp_app]
```

Comment:
Let's rename this to `:nebulex_ecto`

Reply:
Better from:
```ex
...
Application.put_env(:ecto_nebulex, NebulexEcto.TestRepo, user: "invalid")

defmodule NebulexEcto.TestRepo do
  use Ecto.Repo, otp_app: :ecto_nebulex, adapter: NebulexEcto.TestAdapter
...
```
To:
```ex
...
Application.put_env(:nebulex_ecto, NebulexEcto.TestRepo, user: "invalid")

defmodule NebulexEcto.TestRepo do
  use Ecto.Repo, otp_app: :nebulex_ecto, adapter: NebulexEcto.TestAdapter
...
```
To match the whole project definitions.

Done